<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sharikDialog</class>
 <widget class="QDialog" name="sharikDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Параметры первой камеры</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>330</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Широта видеокамеры:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Долгота видеокамеры:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Высота видеокамеры:   -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Азимут:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Угол места:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Число пикселей по горизонтали:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Число пикселей по вертикали:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Расположение шарика по оси абцис:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Расположение шарика по оси ординат:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Размер по горизонатли:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Размер по вертикали:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ширина поля зрения:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Высота поля зрения:  -</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Загрузить параметры первой камеры</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Параметры второй камеры</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>330</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Широта видеокамеры:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Долгота видеокамеры:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Высота видеокамеры:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Азисут:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Угол места:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Число пикселей по горизонтали:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Число пикселей по вертикали:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Расположение шарика по оси абцис:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Расположение шарика по оси ординат:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Размер по горизонатли:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Размер по вертикали:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ширина поля зрения:  -</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Высота поля зрения:  -</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Загрузить параметры второй камеры</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Результаты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Широта:  -</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Долгота:   -</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Высота:   -</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ширина:  -</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Высота:  -</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Рассчитать результат</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
